.github/workflows/ontology_diff.yml

```diff
@@ -0,0 +1,135 @@
+name: 'Create OAK diffs on Pull requests'
+
+on:
+    # Triggers the workflow on pull request events for the master branch
+    pull_request:
+        branches: [ master ]
+
+    # Allows you to run this workflow manually from the Actions tab
+    workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build_branch:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.5
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build ontology
+        run: cd src/ontology; make IMP=FALSE PAT=FALSE MIR=FALSE hp.obo
```

Comment:
I have updated this action to use `hp.obo` instead of `hp-edit.obo`. This is so that we can seamlessly use this not only for diffs of the edit file, but also for release diffs later.

Reply:
It is possible we will end up using hp-simple.obo, but this will be my problem, not yours :)